src/solver/src/parameter_user.C

```diff
@@ -35,6 +35,7 @@
 #include "libmesh/parsed_function.h"
 #include "libmesh/parsed_function_parameter.h"
 
+
```

Comment:
HA! Totally missed that after I was screwing around to figure what was going on with the `ParsedFunction` business.